packages/app/src/reducers/emails.js

```diff
@@ -0,0 +1,19 @@
+import * as ActionTypes from '../actions/types'
+
+const initialState = { emailLogs: [] }
+
+const EmailReducer = (state = initialState, action) => {
+  switch(action.type) {
+    case ActionTypes.RECEIVE_EMAIL_LOG:
+      const emailLogs = [...state.emailLogs, action.emailLog]
+      return Object.assign({}, state, { emailLogs })
+    case ActionTypes.RESET_EMAIL_LOGS:
+      return Object.assign({}, state, { emailLogs: [] })
+    case ActionTypes.RECEIVE_TEST_EMAIL_STATUS:
```

Comment:
In the context of the store this should be related to the action, maybe simply `enable` instead of `testEmailSent`?

Reply:
Replaced with
```js
    case ActionTypes.RECEIVE_EMAIL_FORM_ENABLED:
      return Object.assign({}, state, { enabled: action.enabled })
```